src/backend/aspen/api/schemas/samples.py

```diff
@@ -67,7 +67,7 @@ def get(self, key: Any, default: Any = None) -> Any:
         return default_response
 
 
-class SampleResponseSchema(BaseResponse):
+class SampleResponse(BaseResponse):
```

Comment:
Putting Schema in the name of classes imported from a `schema` module is starting to feel redundant.